.github/workflows/pull_request.yml

```diff
@@ -136,9 +141,11 @@ jobs:
         run: |
           course_name=$(head -n 1 _website.yml | cut -d'"' -f 2| tr " " "-")
           website_link=$(echo "https://htmlpreview.github.io/?https://raw.githubusercontent.com/$GITHUB_REPOSITORY/preview-${{ github.event.pull_request.number }}/docs/index.html")
-          echo ::set-output name=website_link::$website_link
-          echo ::set-output name=time::$(date +'%Y-%m-%d')
-          echo ::set-output name=commit_id::$GITHUB_SHA
+          docs_link=$(echo "https://github.com/$GITHUB_REPOSITORY/raw/preview-${{ github.event.pull_request.number }}/website-preview.zip")
```

Comment:
can we name this something more descriptive, like "zip_file" ? docs_link makes it sound like its gonna take you to documentation

Reply:
Could we also break this chunk up into "# Store variables" and "Print to log" for readability?

Reply:
Oh actually the "echo to log" can't really be separated the `echo "zip_link=$docs_link" >> $GITHUB_OUTPUT` aren't echoing to log, the are saving the variables to the output and have to be done in the same step.